gradle.properties

```diff
@@ -1 +1,2 @@
+org.gradle.jvmargs=-Xmx4G
```

Comment:
Did you test with lower amounts of memory? 1G is enough for Minecraft mods, for example, or 2G for Architectury.